aws/sdk/sdk-external-types.toml

```diff
@@ -1,21 +1,30 @@
 # These are the allowed external types in the `aws-sdk-*` generated crates, checked by CI.
 allowed_external_types = [
-    "aws_credential_types::*",
-    "aws_http::*",
-    "aws_runtime::*",
-    "aws_smithy_async::*",
-    "aws_smithy_http::*",
-    "aws_smithy_runtime::*",
-    "aws_smithy_runtime_api::*",
     "aws_smithy_types::*",
+    "aws_credential_types::*",
     "aws_types::*",
-
-    "http::header::map::HeaderMap",
-    "http::header::value::HeaderValue",
+    "aws_smithy_runtime_api::*",
+    "aws_smithy_runtime::*",
```

Comment:
Since this will be an unstable crate, we should be careful about what gets allowed here.

Reply:
oh good catch, bad copy-paste by me. It's actually unused.